bin/bundle

```diff
@@ -1,3 +1,105 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
```

Comment:
Are all these from regenerating bin stubs or something?

Reply:
Ugh, yes, Rails really wants me to regenerate them. I'll undo it for this PR.